backend/digitales_warten.py

```diff
@@ -0,0 +1,26 @@
+from app import app, db
+from models.slot import Slot
+from models.queue import Queue
+
+@app.route('/')
+def hello_world():
+    return 'Hey, we have Flask in a Docker container!'
+
+@app.route('/queue', methods=['POST'])
+def create_queue():
+    queue = Queue()
+    db.session.add(queue)
+    db.session.commit()
+    return str(queue.id)
+
+@app.route('/slot/<queue_id>', methods=['POST'])
```

Comment:
I know nothing about web-stuff, but this path look strange. having /slot/42 looks like 42 is the id of the slot. I would have expected something like /queue/42/slot

Reply:
I know nothing about web-stuff either, so I just used the first thing I thought of. We should have an API meeting with frontend about that, they probably know more about it :)

Reply:
Done